### ISSUE TITLE
Dusty Old Chest (fizzBuzz.js) - Bug with FAILED messages

### ISSUE THREAD

**Issue by TomJGooding** [TwilioQuest/twilioquest-javascript#11]
**There seems to be a bug with the `** FAILED **` messages for the Dusty Old Chest (i.e. fizzBuzz.js), where the result provided from testing the user's code is actually incorrect, but more importantly the "correct" result it advises is wrong! This could be very confusing for people struggling to solve this!**

I discovered this bug after I somehow forgot the all-important ` == 0` after the remainder operators!

``` javascript
const num = process.argv[2];
if (num % 15) console.log('JavaScript');
else if (num % 3) console.log('Java');
else if (num % 5) console.log('Script');
else console.log(num);
```

This returns the message...
> While testing your script, we passed in "0" but got "JavaScript" instead of "4". 

... which is confusing for a few reasons!
- 0 is divisible by both 3 and 5, so actually "JavaScript" should be the correct result - not "4"
- When testing my code (which I should have done in the first place!) with `node fizzBuzz.js 0`, the actual result is "0" - not "JavaScript"

I tried other tests with different mistakes in the code and discovered more confusing  `** FAILED **` messages. For example this code...

``` javascript
const num = process.argv[2];
if (num == 0) console.log('JavaScript');
else if (num % 3 == 0) console.log('Java');
else if (num % 5 == 0) console.log('Script');
else console.log(num);
```
... returns this message:

> While testing your script, we passed in "3" but got "Java" instead of "JavaScript".

The correct result for 3 should be "Java", which is actually what the above code returns when testing with `node fizzBuzz.js 3`

**Comment by MarvelProgramming** [comment]
Fixed this in #13

The issue ended up being the string that was being interpolated for the error message. As an example, it'd pass in '3' as the input but say '0' in the error message.

**Comment by rmkubik** [comment]
We shipped #13 which resolved this issue!
